src/tgconf_beamviewer/panels/beamviewerwidget.py

```diff
@@ -4,8 +4,14 @@
 from functools import wraps, partial
 
 from taurus import Attribute, Device
-from taurus.qt.qtgui.panel import TaurusWidget
-from taurus.qt import QtGui, QtCore
+try:
```

Comment:
I gather this is about supporting both Taurus v.4 and older versions at the same time.

It's not very obvious though, what about adding a short comment that explains it, so that eventually (when older versions are no longer in use anywhere) this can all be removed? E.g. "# Support Taurus versions older than 4".